testhelpers/image_privileges.go

```diff
@@ -0,0 +1,35 @@
+package testhelpers
+
+import (
+	"regexp"
+	"strings"
+)
+
+var urlRegex = regexp.MustCompile(`v2\/(.*)\/(?:blobs|manifests|tags)`)
+
+type ImagePrivileges struct {
+	readable bool
+	writable bool
+}
+
+func NewImagePrivileges(imageName string) (priv ImagePrivileges) {
```

Comment:
Could we add a docs comment here? Keying the permissions off the image name (while convenient) wasn't immediately obvious to me.

Reply:
Added in this [commit](https://github.com/buildpacks/imgutil/pull/125/commits/2486b56700226982aa19e6e383bba488b80e0394)